k8s/openebs-storagepoolclaims.yaml

```diff
@@ -0,0 +1,19 @@
+---
+apiVersion: openebs.io/v1alpha1
+kind: StoragePoolClaim
+metadata:
+  name: cstor-pool-default-0.7.0
+  annotations:
+    cas.openebs.io/create-pool-template: cstor-pool-create-default-0.7.0
+    cas.openebs.io/delete-pool-template: cstor-pool-delete-default-0.7.0
+spec:
+  name: cstor-pool-default-0.7.0
+  type: openebs-cstor
+  maxPools: 3
+  poolSpec:
+    poolType: striped
+    cacheFile: /tmp/pool1.cache
+    overProvisioning: false
+  disks:
+    diskList:
+      - 
```

Comment:
It will be better if we put a comment here about how the disk to be added with an example ?